lib/workers/repository/extract/extract-fingerprint-config.ts

```diff
@@ -55,13 +55,9 @@ export function generateFingerprintConfig(
   config: RenovateConfig
 ): FingerprintExtractConfig {
   const managerExtractConfigs: WorkerExtractConfig[] = [];
-  let managerList: Set<string>;
-  const { enabledManagers } = config;
-  if (enabledManagers?.length) {
-    managerList = new Set(enabledManagers);
-  } else {
-    managerList = new Set(allManagersList);
-  }
+  const managerList: Set<string> = new Set(
```

Comment:
```suggestion
  const managerList = new Set(
```
i think we can drop the explicit type here